data/homepage.yaml

```diff
@@ -11,6 +11,24 @@ bullhorn:
   label: Email address
   placeholder: Sign me up for Ansible community news
   button: Subscribe
+events:
+  title: Events
+  intro: Join Ansible events, in person or online, to share and learn about automation.
+  url: https://forum.ansible.com/c/events/
+  view_calendar: View all Ansible events
+  events_list:
+    event_one:
```

Comment:
Can we add a URL property here to link to the specific forum event? People should be able to go hype about it :). This could also point to eventbrite etc, or a blog post about the upcoming event - we just need to give people a path to move forward if they're interested/

Reply:
good point. done, including the other one.